Source/Resources/Content/AadIam/PasswordResetPolicies/Configuration.json

```diff
@@ -1,6 +1,7 @@
 {
   "customHelpdeskEmailOrUrl": "",
   "customizeHelpdeskLink": false,
+  "description":"What is it: Defines Azure's password reset policy and which Azure AD groups it applies to. \n Simeon Notes: This allows users to reset their password or unlock their Azure account. Simeon's baseline applies the password reset policy to members of the group 'BUILTIN - Microsoft 365 Users'. \n Learn More: https://docs.microsoft.com/en-us/azure/active-directory/authentication/concept-sspr-howitworks",
```

Comment:
**What does this do?**
Defines Azure's password reset policy and specifies the Azure AD groups it applies to. This allows users to reset their password or unlock their Azure account. The baseline applies the password reset policy to members of the group 'Baseline - Microsoft 365 Users'.  

@metamenos what are the password reset policies included in this setting? I see that email option is false, and mobile app code is true, but want to clarify what that means 

**Why should you use this?**
If you want to ensure that users can securely recover and reset their passwords. 

**What is the end user impact?**
Users will be able to securely recover and reset their passwords.

@metamenos probably need to include something about the manner in which they're able to do so (based on your response to the above question)

**Learn more**
[How it works: Azure AD self-service password reset](https://docs.microsoft.com/en-us/azure/active-directory/authentication/concept-sspr-howitworks)

Reply:
That means that if someone tries to reset their password they will not be able to have the pw reset link emailed to them. Looks like the only only options would be Mobile app code (the authenticator app) or Mobile phone sms. 

![image](https://user-images.githubusercontent.com/68136026/91115686-55d4c100-e650-11ea-880d-bdd2b72fb8bf.png)

Reply:
@metamenos totally makes sense. I changed it accordingly